<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixClass</class>
 <widget class="QWidget" name="MatrixClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="StartBtn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>设定线程数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ThreadnumEd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>串行程序计算时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="serialTimeEd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>并行程序计算时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="paralleltimeEd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>加速比</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="speedupEd"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>在线性代数中， LU分解(LU Factorization)是矩阵分解的一种，
可以将一个矩阵分解为一个下三角矩阵和一个上三角矩阵的乘积
（有时是它们和一个置换矩阵的乘积）。
LU分解主要应用在数值分析中，
用来解线性方程、求反矩阵或计算行列式。</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
